### PR TITLE
atmos jumpsuits now have the same protections as engi jumpsuit

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

atmos jumpsuit now have the same stats as engi suit, giving it minor protection from rads, fire and acid
note that fire and acid arent stats that affect the owner, they just cause the jumpsuit to burn down/be destroyed by acid slower

## Why It's Good For The Game

departments usually have consistent protections across their jumpsuits, like the medical suits having bio protection and lower permeability, security having 10 melee armor, so its nice for atmos suits to have fire protection like engi suits, especially when atmos is the job that literally works with fire

## Changelog
:cl:
tweak: atmos jumpsuit now has the same armor values as engi jumpsuit
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
are desc comments needed? there was a pr that adds "hints" of what protection any piece of clothing provides

**Comment by ExcessiveUseOfCobblestone** [comment]
if they are SUPPOSE to be equivalent it should be a define.

**Comment by Kaffe-work** [comment]
they are supposed to be different because they deal with different problems, ie engine malfunction and plasmafires. one could argue that engi hardsuit could have 100% rad protect, because using a radsuit when engine is breaking is suicide due to pressures.

File: code/modules/clothing/under/jobs/engineering.dm [hunk 1]
```diff
@@ -22,15 +22,16 @@
 	fitted = FEMALE_UNIFORM_TOP
 
 /obj/item/clothing/under/rank/engineering/atmospheric_technician
-	desc = "It's a jumpsuit worn by atmospheric technicians."
+	desc = "It's a jumpsuit worn by atmospheric technicians. It has minor protection from fire."
 	name = "atmospheric technician's jumpsuit"
 	icon_state = "atmos"
 	item_state = "atmos_suit"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 60, "acid" = 20)
```
Comment (ATH1909) [inline code/modules/clothing/under/jobs/engineering.dm:29]: why not just move this line (or, well, the line it's copying) to /obj/item/clothing/under/rank/engineering?

**Comment by Fikou** [comment]
> they are supposed to be different because they deal with different problems, ie engine malfunction and plasmafires. one could argue that engi hardsuit could have 100% rad protect, because using a radsuit when engine is breaking is suicide due to pressures and temperatures.

lol

File: code/modules/clothing/under/jobs/engineering.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/clothing/under/jobs/engineering.dm:29]: other jumpsuits didnt do that so i didnt want to do that

**Comment by Fikou** [comment]
> if they are SUPPOSE to be equivalent it should be a define.

idk if armor values should be defines

**Comment by Kaffe-work** [comment]
ah yes, I'm actually just fucking mentally slow without reading comprehension, top kek

**Comment by pireamaineach** [comment]
You should probably make sure this also is applied to atmos plasmaman suits if they share the same values

**Comment by kopoba** [comment]
i was touching fire protection and it turns out this isnt about protection at all it just about how fast clothes burn down and you will get same amount of burn damage from fire with "fire" = 0 and "fire" = 100

**Comment by ExcessiveUseOfCobblestone** [comment]
> 
> 
> > if they are SUPPOSE to be equivalent it should be a define.
> 
> idk if armor values should be defines

you would define the entire list.

```DM
#define ARMOR_ENGINEERING list(melee ...)
```
vs.
```DM
#define ARMOR_ENGINEERING_MELEE 0
#define ARMOR_ENGINEERING_FIRE 10
...
```

If we're going to have to get this PR every time someone changes the value of one then I don't see why it shouldn't.

**Comment by Fikou** [comment]
@ExcessiveUseOfCobblestone what do

**Comment by ExcessiveUseOfCobblestone** [comment]
no more armor PRs unless it's reworking armor to be understandable.

**Pull request merged**